util/helm.go

```diff
@@ -122,3 +122,8 @@ func CopyDir(src, dst string) error {
 	filepath.Walk(src, walker)
 	return failure
 }
+
+// ExpandChartPath - Expand chart path from name and home directory
+func ExpandChartPath(dir, chartName string) string {
```

Comment:
I was just thinking last night that I really wanted a library of functions to do this for me. My code is totally littered with filepath.Join calls for the same 4 things: base directory (where config.yaml is), workspace charts, charts in a repo, and directory where repos are.